projekty/tic_tac_toe_nowe_funkcje.py

```diff
@@ -0,0 +1,115 @@
+class Board:
+
+    def __init__(self):
+        # tworzenie pustej tablicy
+        self.board = [' ' for _ in range(10)]
+
+    def __str__(self):
+        return (
+                f'{self.board[7]}' + ' | ' + f'{self.board[8]}' + ' | ' + f'{self.board[9]}\n' +
+                f'_'*9 + '\n'
+                         f'{self.board[4]}' + ' | ' + f'{self.board[5]}' + ' | ' + f'{self.board[6]}\n' +
+                f'_'*9 + '\n'
+                         f'{self.board[1]}' + ' | ' + f'{self.board[2]}' + ' | ' + f'{self.board[3]}\n'
+        )
+
+    def add_to_board(self, place: int, marker: str, player_set: set):
+        if place in player_set:
+            raise ValueError('Error in Board class add_to_board. Wrong number')
+        elif place not in player_set:
+            self.board.pop(place)
+            self.board.insert(place, marker)
+
+
+class Player:
+    def __init__(self, marker):
+        self.marker = marker
+        self.player_set = set()
+
+    def __str__(self):
+        return f'Player {self.marker}'
+
+    def place(self):
+        place_to_choose = int(input('Select number 1-9 to place marker there: '))
+        if place_to_choose in range(1, 10):
+            return place_to_choose
+        else:
+            raise ValueError
+
+
+def win_check(player_set, player):
```

Comment:
sterowanie lepiej zrobić typach bool
```suggestion
    def win_check(self, player: Player):
        for w_set in self.win_set:
            if player.player_set.intersection(w_set) in self.win_set:
                return True
        return False
```